aws/db.tf

```diff
@@ -80,3 +80,17 @@ module "db" {
 
   tags = local.tags
 }
+
+data "aws_route53_zone" "private_zone" {
```

Comment:
No need to data the resource. We create it inside the module.
https://github.com/datagrok-ai/tf-module-datagrok-core/blob/19fcab1827b86a67811f73c81c90e53372cf68d5/aws/lb.tf#L222-L229